<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2104</width>
    <height>984</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="codeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="plainText">
     <string>
cout &lt;&lt; &quot;Hola&quot;;

int a = 1 + 2;
int b = 4 / 2;
int c = 5 * 2;
int d = 43 - 2;

long e = 2 + 2;
long f = 4000000 / 2;
long g = 5432 * 2;
long h = 43534 - 2423;

float i = 1 + 2;
float j = 4100 / 593;
float m = 23 * 2;
float n = 433 - 24;

double l = 16.65 + 27.34;
double o = 487.54 / 245.67;
double p = 52.43 * 21.54;
double q = 35431.3 - 2433.1;

int r;
float s;
double t;
long u;
char v;

int r = 1;
float s = 1.3;
double t = 1.554;
long u = 1234455;
char v = &quot;a&quot; + &quot;a&quot;;
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="appLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>1481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="ramLiveView">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>10</y>
      <width>411</width>
      <height>861</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Console</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="terminalOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>1481</width>
      <height>151</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2104</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext_Line"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVuild"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionNext_Line"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_folder-open-storage-file-organize_3209339.png</normaloff>../../Descargas/iconfinder_folder-open-storage-file-organize_3209339.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_158-Save_2123968.png</normaloff>../../Descargas/iconfinder_158-Save_2123968.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_circle_close_cancel_exit_quit_delete_4831037.png</normaloff>../../Descargas/iconfinder_circle_close_cancel_exit_quit_delete_4831037.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_icon-play_211876.png</normaloff>../../Descargas/iconfinder_icon-play_211876.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_interface-solid-multimedia-stop-button-1_6127256.png</normaloff>../../Descargas/iconfinder_interface-solid-multimedia-stop-button-1_6127256.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNext_Line">
   <property name="icon">
    <iconset>
     <normaloff>../../Descargas/iconfinder_right_4_1_4829859.png</normaloff>../../Descargas/iconfinder_right_4_1_4829859.png</iconset>
   </property>
   <property name="text">
    <string>Next Line</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
